### ISSUE TITLE
Missing permissions in documentation and example script.

### ISSUE THREAD

**Issue by maxwiks** [MicrosoftDocs/powerapps-docs#1120]
The examples provided in this page for creating a user in SQL are missing database permissions for **VIEW DATABASE STATE**. This permission is described in docs [here](https://docs.microsoft.com/en-us/power-platform/admin/replicate-data-microsoft-azure-sql-database)

**Comment by phecke** [comment]
@maxwiks 
Are you saying that the paragraph in the doc should read:

The database user, specified in the data export connection string, must have the proper create and alter permissions on the target database. At minimum these include: CRTB, CRTY, CRVW, CRPR, **VWDS**, and ALUS. For more information, see Permissions (Database Engine).

**Comment by maxwiks** [comment]
@phecke Exactly. The same thing goes for the example SQL script just underneath. It should include VIEW DATABASE STATE.

**Comment by KendalBond007** [comment]
Thanks for your feedback. The document was updated as described by phecke (27 Mar 22), with a most recent publish date of 4 Sep 2022. If you're still experiencing the issue described here, please reopen or create a new issue. Closing this issue.
